Stage 3/script.js

```diff
@@ -0,0 +1,58 @@
+let player1Score = 0;
+let player2Score = 0;
+
+function startQuiz() {
+  const player1Name = document.getElementById("player1Name").value;
+  const player2Name = document.getElementById("player2Name").value;
+
+  document.getElementById(
+    "player1"
+  ).textContent = `${player1Name}: ${player1Score} points`;
+  document.getElementById(
+    "player2"
+  ).textContent = `${player2Name}: ${player2Score} points`;
+}
+
+function updateScore(playerNumber, isCorrect) {
+  const player1PointsInput = document.getElementById("player1Points");
+  const player2PointsInput = document.getElementById("player2Points");
+
+  if (playerNumber === 1) {
+    if (isCorrect) {
+      player1Score += 1;
+    } else {
+      player2Score += 1;
+    }
+    player1PointsInput.value = player1Score;
```

Comment:
This only updates player1, and therefore it won't update the player2Score, so maybe we should move the player1PointsInput and player2PointsInput value update outside the if statement 😄